src/applications/personalization/profile/tests/components/contact-information/ContactInformation.delete-address.unit.spec.jsx

```diff
@@ -111,7 +111,7 @@ async function testSlowFailure(addressName) {
 
   server.use(...mocks.transactionFailed);
 
-  await wait(20);
+  await wait(1500);
```

Comment:
I did have to push up these wait calls because they kept failing in CI/CD. There are talks about using a different version of the VA Profile api that wouldn't rely on this 'transactional' system so 🤞🏼 in the future I can remove these wait calls altogether.

Reply:
thanks for looking into it! 💯

Reply:
thanks for looking into it! 💯